### ISSUE TITLE
crash when lauching the game.

### ISSUE THREAD

**Issue by ceifadorcinixtru** [jakeb072001/TerraCraft#13]
Im getting this error when i try to launch this mod in my modpack - Caused by: org.spongepowered.asm.mixin.injection.throwables.InjectionError: Critical injection failure: Redirector hideElytra(Lnet/minecraft/class_1799;Lnet/minecraft/class_1792;Lnet/minecraft/class_4587;Lnet/minecraft/class_4597;ILnet/minecraft/class_1309;FFFFFF)Z in terramine.mixins.json:client.render.ElytraLayerMixin from mod terramine failed injection check, (0/1) succeeded. Scanned 1 target(s). Using refmap terramine-refmap.json
		at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.postInject(InjectionInfo.java:468)
		at org.spongepowered.asm.mixin.transformer.MixinTargetContext.applyInjections(MixinTargetContext.java:1384)

Fabric version - 0.14.24
Minecraft - 1.19.2
Mod version - 1.7.0

**Comment by ceifadorcinixtru** [comment]
It looks like its incompatible with origins mod.

**Comment by jakeb072001** [comment]
I'll have a look, from what I can tell with this log I need to make the hideElytra method @Unique so it doesn't conflict with origins method of the same name.

I'll try to test this a bit later and post a fix here if it works

**Comment by ceifadorcinixtru** [comment]
i think the same thing, ill be glad of testing the next update.

**Comment by jakeb072001** [comment]
Alright, that wasn't the issue. I did get it working though. Basically apoli uses redirect for their elytra mixin and so was I and two redirects is a big bad. I have improved my code to be a lot more compatible so its possible I've fixed other conflicts with various other mods as well.

Anyway, here's the fix, give it a shot, I have a bit of work still before properly releasing the fix.
[Uploading update.zip…]()
